src/explore-education-statistics-common/src/modules/find-statistics/queries/tableBuilderQueries.ts

```diff
@@ -0,0 +1,22 @@
+import tableBuilderService from '@common/services/tableBuilderService';
+import { createQueryKeys } from '@lukemorales/query-key-factory';
+
+const tableBuilderQueries = createQueryKeys('tableBuilder', {
+  getGeoJson(
+    releaseId: string,
+    dataBlockParentId: string,
+    boundaryLevelId: number,
+  ) {
+    return {
+      queryKey: ['geoJson', releaseId, dataBlockParentId, boundaryLevelId],
```

Comment:
Don't need the `geoJson` string as part of the query key. `createQueryKeys` will automatically include the method name into the query key. It will automatically generate something like:

```ts
['tableBuilder', 'getGeoJson', releaseId, dataBlockParentId, boundaryLevelId]
```

When we don't use `createQueryKeys`, we need to include some unique string (like the method name) to provide enough uniqueness to the query key. 

Currently, we're not using `createQueryKeys` in the public frontend, but now that we've started using this in common, we'll probably want to convert the existing queries to use it at some point down the line.